src/ess.jl

```diff
@@ -211,8 +211,15 @@ function ess(
     # estimate the effective sample size and rhat
     ess, rhat = ess_rhat(_chains.value.data; kwargs...)
 
+    # Calculate ESS/minute if available
+    dur = wall_duration(chains)
+
     # convert to namedtuple
-    nt = merge((parameters = names(_chains),), (ess = ess, rhat = rhat))
+    nt = if ismissing(dur)
```

Comment:
I assume that similar to `nothing` it is sometimes better to check `=== missing` instead of `ismissing` (I guess above it does not matter since it does not affect the output type):
```suggestion
    nt = if dur === missing
```